src/accountingservice/Dockerfile

```diff
@@ -26,6 +26,7 @@ COPY --from=publish /app/publish .
 USER root
```

Comment:
would it make sense to change the user here, as now no root is required?

Reply:
It is needed because is the user with permission to change permissions over the other files.